theme.ts

```diff
@@ -1,3 +1,15 @@
-import type { MantineThemeOverride } from '@mantine/core';
+import { MantineThemeOverride } from '@mantine/core';
+import { DEFAULT_THEME } from '@mantine/core';
 
-export const theme: MantineThemeOverride = {};
+export const theme: MantineThemeOverride = {
+  primaryColor: 'pink',
+  defaultRadius: 'xl',
+  fontFamily: `Inter,${DEFAULT_THEME.fontFamily}`,
+  fontSizes: {
+    xs: 16,
+    sm: 18,
+    md: 20,
+    lg: 22,
+    xl: 24,
+  },
```

Comment:
I think just changing fontSizes doesn't suffice, it's important to adjust other sizes too
Mantine's system is not adaptive, for every t-shirt size you need to tweak all related theme variables and maybe even each component individual sizes, because e.g. in the text-input right now the placeholder looks too big

alternative way is to not override default theme sizes, but rather set `<MantineButton size="md">` or higher and make use of Mantine preconfigured sizes, actually I like this idea even more, it requires wrapping each of Mantine's component (which we are doing anyway) and hardcoding this value to the same t-shirt size (not difficult, just important not to forget to do it for all wrapped components)